test/tester2000.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { token, prefix, owner } = require('./auth.js');
+const { Client, Options, Intents } = require('../src');
+
+// eslint-disable-next-line no-console
+const log = (...args) => console.log(process.uptime().toFixed(3), ...args);
+
+const client = new Client({
+  intents: Intents.ALL,
+  makeCache: Options.cacheWithLimits({
+    Message: 10,
+    Presence: 10,
+  }),
+});
+
+client.on('debug', log);
+client.on('ready', () => {
+  log('READY', client.user.tag, client.user.id);
+});
+client.on('rateLimit', log);
+client.on('error', console.error);
+
+const commands = {
+  eval: message => {
+    if (message.author.id !== owner) return;
+    let res;
+    try {
+      res = eval(message.content);
+      if (typeof res !== 'string') res = require('util').inspect(res);
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.error(err.stack);
+      res = err.message;
+    }
+    message.channel.send(res, { code: 'js' });
```

Comment:
This no longer works.

Reply:
Bleh, tell whoever made tester1000 about it.